config/locales/en.yml

```diff
@@ -184,6 +184,15 @@ en:
         option_no:
           label: "No"
       custom_select_error: Select whether you can offer care
+    offer_community_support:
+       title: Can you offer community support?
+       hint: For example distributing food or helping homeless people.
+       options:
+         option_yes:
+           label: "Yes"
+         option_no:
+           label: "No"
+       custom_select_error: Select which services you can provide
```

Comment:
You might want a more specific error message here, but no worries if you're going to update the content later